init_scripts/vep105-init.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+export PROJECT="$(gcloud config get-value project)"
+export VEP_CONFIG_PATH="$(/usr/share/google/get_metadata_value attributes/VEP_CONFIG_PATH)"
+export VEP_REPLICATE="$(/usr/share/google/get_metadata_value attributes/VEP_REPLICATE)"
+export VEP_BUCKET=hail-${VEP_REPLICATE}-vep
+export ASSEMBLY=GRCh38
+export VEP_DOCKER_IMAGE=us-central1-docker.pkg.dev/broad-mpg-gnomad/images/vep105
+
+mkdir -p /vep_data/loftee_data
+mkdir -p /vep_data/homo_sapiens
+
+# Install docker
+apt-get update
+apt-get -y install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common \
+    tabix
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+apt-get update
+apt-get install -y --allow-unauthenticated docker-ce
+
+
+gsutil -u $PROJECT cat gs://${VEP_BUCKET}/loftee-beta/${ASSEMBLY}.tar | tar -xf - -C /vep_data/ &
+#gsutil -u $PROJECT cat gs://${VEP_BUCKET}/Plugins.tar /vep_data/Plugins.tar | tar -xf - -C /vep_data
```

Comment:
Is this line not needed? If it isn't needed let's remove it

Reply:
We don't need it, because we copied Plugins in the docker image.

Reply:
and the Plugins from the bucket was out-dated, that's why I quoted it.